core/templates/dev/head/domain/exploration/SolutionObjectFactory.js

```diff
@@ -68,8 +68,7 @@ oppia.factory('SolutionObjectFactory', [
       var explanation = (
         $filter('convertToPlainText')(this.explanation));
       return (
-        '[' + solutionType + ' solution is ' + correctAnswer + '] ' +
-        explanation);
+        solutionType + ' solution is ' + correctAnswer + '. ' + explanation);
```

Comment:
Item selection is enclosed with [...] so having quotes around that shouldn't be an issue. It's true that text input strings can include quotes but that's not common. So I think putting quotes around just the correct answer is fine.